app.py

```diff
@@ -44,7 +44,14 @@ def get_users_profile():
         return jsonify([e.serialize() for e in users])
     except Exception as e:
             return(str(e))
-      
-      
+
+@app.route('/speech_questions')
+def get_speech_questions():
+    try:
+        users = SpeechQuestion.query.all()
+        return jsonify([e.serialize() for e in users])
+    except Exception as e:
+            return(str(e))
+
```

Comment:
Do you think we'll need to get speech questions based on difficulty, as opposed to all of them at once?

Reply:
Yes, there should be methods in sqlalchemy that allow us to filter speech questions based on level of difficulty